### DISCUSSION TITLE
Override "associate" callback handler

### DISCUSSION THREAD

**Discussion by holtgrewe** [fastapi-users/fastapi-users#1334] [Q&A]
Maybe I'm using fastapi-users in a non-intended way, but here it goes.

I let the user go to the associate handler for cookie authentication. The browser nicely goes to the upstream authentication provider and is associated. However it ends up in the following location that renders a JSON document.

- `/api/v1/auth/external/associate/cookie/AUTH/callback?...`

(How) can I override the behaviour and have the user be redirect to the profile page again?

Thanks,
Manuel

**Comment by hgalytoby** [comment]
I'm not sure if I understand your question correctly, but I would like to offer the following suggestions:

https://github.com/fastapi-users/fastapi-users/discussions/906

https://github.com/fastapi-users/fastapi-users/discussions/1154

**Comment by gromdimon** [comment]
#1154 solved the issue. Thanks @hgalytoby. @holtgrewe probably can close this discussion.
